### PR TITLE
pylint ignore

### PR DESCRIPTION
**Pull request by peasant98**
addressing issue from https://github.com/HIRO-group/robotic_skin/issues/43; ignoring pylint for two files.

### CODE REVIEW

File: examples/07_read_vcnl4040.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 """
 Read vcnl4040 sensor example
 """
+
```
Comment (watakandai) [inline examples/07_read_vcnl4040.py]: oh no need to modify this. Could you just remove these two files? We do not need it anymore

**Review by watakandai — CHANGES_REQUESTED** [review]

File: examples/07_read_vcnl4040.py (hunk 1 above)
Comment (peasant98) [reply examples/07_read_vcnl4040.py]: Ah ok, will take care of that.

**Review by krishnachaitanya7 — CHANGES_REQUESTED** [review]
Are the files deleted permanently?

**Comment by peasant98** [comment]
yes

**Pull request merged**